Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.13
+FROM alpine:3.14
```

Comment:
Could you revert it? We should have a dedicated PR.

Reply:
Done. Will raise a separate PR.